### DISCUSSION TITLE
Bot is not playing local wav sound file

### DISCUSSION THREAD

**Discussion by CreepySkyYT** [Rapptz/discord.py#6417] [Q&A]
Hey i have coded a discord bot and i want to let him join a channel when he goes online and the he should play music. Also i would to trigger it with !boot but i don't know how to code this my code to now is

    if message.content.startswith('!boot'):
        voice = await message.author.voice.channel.connect()
        voice.play(discord.FFmpegPCMAudio('./res/J.A.R.V.I.S. Boot.wav'))

But python can't read it and sais that ffmpeg is not founded but ffmpeg is installed. What should I
 do now?

![Screenshot 2021-02-12 155526](https://user-images.githubusercontent.com/78969104/107783730-de543480-6d4a-11eb-9ab3-338cab331244.png)

**Comment by tk421bsod** [comment]
You'll need to either add FFmpeg to your PATH (see [this stack exchange post](https://video.stackexchange.com/questions/20495/how-do-i-set-up-and-use-ffmpeg-in-windows) for more information) or put the exe in the same directory as your bot.
